src/Microsoft.SqlTools.ServiceLayer/QueryExecution/QueryExecutionService.cs

```diff
@@ -165,7 +166,85 @@ public void InitializeService(ServiceHost serviceHost)
             Func<string, Task> queryCreateFailureAction = message => requestContext.SendError(message);
 
             // Use the internal handler to launch the query
-            return InterServiceExecuteQuery(executeParams, requestContext, queryCreateSuccessAction, queryCreateFailureAction, null, null);
+            return InterServiceExecuteQuery(executeParams, null, requestContext, queryCreateSuccessAction, queryCreateFailureAction, null, null);
+        }
+
+        /// <summary>
+        /// Handles a request to execute a string and return the result
+        /// </summary>
+        internal Task HandleSimpleExecuteRequest(SimpleExecuteParams executeParams,
+            RequestContext<SimpleExecuteResult> requestContext)
+        {
+             ExecuteStringParams executeStringParams = new ExecuteStringParams
+            {
+                Query = executeParams.QueryString,
+                // generate guid as the owner uri to make sure every query is unique
+                OwnerUri = Guid.NewGuid().ToString()
+            };
+
+            // get connection
+            ConnectionInfo connInfo;
+            if (!ConnectionService.TryFindConnection(executeParams.OwnerUri, out connInfo))
```

Comment:
whats the difference? the connection finding in CreateQuery uses tryfindconnection, so I just copied that.